### PR TITLE
docs: improve example config

### PR DESCRIPTION
**Pull request by willothy**
The example config was inconsistent for me because my `vim.diagnostic.config` likely differs from yours. Here's an update to the example config that manually sets the float config - it should work consistently across user configs now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,24 +47,38 @@ use {
...
         if vim.w.diag_shown then return end
 
         -- open float-win when hovering on a cursor-word.
-        if vim.w.cursor_word ~= '' then
-          vim.diagnostic.open_float()
+        if vim.w.cursor_word ~= "" then
+          local buf = vim.diagnostic.open_float({
+            scope = "cursor",
+            -- Only close the window on InsertEnter and the explicit diagnostic close event
+            close_events = { "InsertEnter", "User MurmurDiagnostics" },
```
Comment (nyngwang) [inline README.md:69]: TIL. I tried very hard to know how to trigger my user event but gave up.

Comment (willothy) [reply README.md:69]: Yep! Learned this pretty recently as well haha

**Comment by nyngwang** [comment]
I will accept this first since I prefer to progress iteratively :) Comments are comments.

**Comment by willothy** [comment]
> I will accept this first since I prefer to progress iteratively :) Comments are comments.

Sounds good :)

**Pull request merged**